core/src/main/java/org/axonframework/serialization/UnknownSerializedType.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.serialization;
+
+/**
+ * Class representing a serialized object of which there is no class available in the current class loader. This
+ * class provides access to the raw underlying data, as well as any format supported by the serializer.
+ *
+ * @since 4.0
+ */
+public class UnknownSerializedType {
+
+    private final Serializer serializer;
+    private final SerializedObject<?> serializedObject;
+
+    /**
+     * Initialize the unknown type, using given {@code serializer} and {@code serializedObject}.
+     *
+     * @param serializer       The serializer attempting to deserialize the object
```

Comment:
Nit: I believe we've got a mixture of either upper case or lower casing `@param`/`@return` descriptions. What should it be?